src/coefficients/coefficient_extensions.hpp

```diff
@@ -19,7 +19,7 @@
 #include "mfem.hpp"
 #include "numerics/expr_template_ops.hpp"
 
-namespace serac {
+namespace serac::mfem_extensions {
 
 /**
  * @brief MakeTrueEssList takes in a FESpace, a vector coefficient, and produces a list
```

Comment:
Should we only do this for the class extensions and code we hope to get back into MFEM?  Do these functions fall under that? they seem like helper functions...

Clearly not from this PR but the documentation case doesn't match the actual function name case (`MakeTrueEssList vs `makeTrueEssList`)

Reply:
Let's talk about this at the meeting tomorrow.